### PR TITLE
Dialog notifier 0.1.0

### PR DESCRIPTION
**Pull request by nkt**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/dialog-notifier/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- DIALOG_WEBHOOK_URL: $DIALOG_WEBHOOK_URL
```
Comment (richard-bogdan-bitrise) [inline steps/dialog-notifier/0.1.0/step.yml]: Inputs should use lowercase letter. please check the step development guide:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md

File: steps/dialog-notifier/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+- notification
+toolkit:
+  go:
+    package_name: github.com/dialogs/bitrise-step-dialog-notifier
+deps:
```
Comment (richard-bogdan-bitrise) [inline steps/dialog-notifier/0.1.0/step.yml]: As I see, your step code does not need the use of git.

File: steps/dialog-notifier/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+summary: |
+  Notify Dialog chat about build status
+description: |
+  Notify Dialog chat about build status
+website: https://github.com/dialogs/bitrise-step-dialog-notifier
+source_code_url: https://github.com/dialogs/bitrise-step-dialog-notifier
+support_url: https://github.com/dialogs/bitrise-step-dialog-notifier/issues
+published_at: 2019-08-29T13:28:07.135529+03:00
+source:
+  git: git@github.com:dialogs/bitrise-step-dialog-notifier.git
```
Comment (richard-bogdan-bitrise) [inline steps/dialog-notifier/0.1.0/step.yml]: currently only http or https is supported for source, please update this.

**Comment by godrei** [comment]
Closing as we did not receive a reply, @nkt pls have a look at our comments, fix the step.yml, tag the new commit in the step repo and share the new version of the step.

**Pull request closed without merging**